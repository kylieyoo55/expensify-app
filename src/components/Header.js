import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = ()=>(
    <div>
        <h1>Expensify</h1>

        <NavLink to={'/'} activeClassName='is-active'>--HOME--</NavLink>
        <NavLink to={'/add'} activeClassName='is-active'>--ADD EXPENSE--</NavLink>
 
        <NavLink to={'/help'} activeClassName='is-active'>--HELP--</NavLink>
        
    </div>
)

export default Header;