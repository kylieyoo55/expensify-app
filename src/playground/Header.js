import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = ()=>(
    <div>
        <h1>HEADER</h1>
        <NavLink to={'/'} activeClassName='is-active'>Home --</NavLink>
        <NavLink to={'/contact'} activeClassName='is-active'>Contact --</NavLink>
        <NavLink to={'/portfolio'} activeClassName='is-active'>Portfolio</NavLink>

        
    </div>
)

export default Header;