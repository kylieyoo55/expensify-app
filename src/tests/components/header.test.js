import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/Header';
import toJSON from 'enzyme-to-json';
//react-test-renderer
//should render Header correctly

test('should render Header correctly',()=>{
const wrapper=shallow(<Header />);
expect(toJSON(wrapper)).toMatchSnapshot();
// expect(wrapper.find('h1').text()).toBe('Expensify');
})

