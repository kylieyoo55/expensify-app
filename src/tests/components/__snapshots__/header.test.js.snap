// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    to="/"
  >
    --HOME--
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/add"
  >
    --ADD EXPENSE--
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    --HELP--
  </NavLink>
</div>
`;
